packages/scratch-gui/test/unit/components/cards.test.jsx

```diff
@@ -36,27 +37,31 @@ describe('Cards component', () => {
     });
 
     test('showVideos=true shows the video step', () => {
-        const component = mountWithIntl(
-            <Cards
-                {...defaultProps()}
-                showVideos
-            />
-        );
-        expect(component.find(ImageStep).exists()).toEqual(false);
-        expect(component.find(VideoStep).exists()).toEqual(true);
+        const {container} = renderWithIntl(<Cards {...defaultProps()} showVideos />);
+        const videoStep = container.querySelector('#video-div');
+        expect(videoStep).toBeTruthy();
+    
+        const imageStep = container.querySelector('img[src="id1 - img"]');
+        expect(imageStep).toBeNull();
     });
 
     test('showVideos=false shows the title image/name instead of video step', () => {
-        const component = mountWithIntl(
-            <Cards
-                {...defaultProps()}
-                showVideos={false}
-            />
-        );
-        expect(component.find(VideoStep).exists()).toEqual(false);
+        const { container } = renderWithIntl(<Cards {...defaultProps()} showVideos={false} />);
+        
+        const videoStep = container.querySelector('#video-div');
+        expect(videoStep).toBeNull();
+    
+        const imageStep = container.querySelector('img');
+        expect(imageStep).toBeTruthy();
 
-        const imageStep = component.find(ImageStep);
-        expect(imageStep.props().image).toEqual('id1 - img');
-        expect(imageStep.props().title).toEqual('id1 - name');
+        const divWithText = [...container.querySelectorAll('div')].find(el => el.textContent.includes('id1 - name'));
+        expect(divWithText).toBeTruthy();
+        expect(divWithText).toHaveTextContent('id1 - name');
```

Comment:
if `divWithText` is truthy wouldn't that mean it has the text context, because we use it in the query selector. Same for the image